src/main/java/Models/User/User.java

```diff
@@ -0,0 +1,70 @@
+package Models.User;
+
+//import java.sql.Date;
+
+import java.util.Arrays;
+
+public class User {
+     private final Integer ID;
+     private final String firstName;
+     private final String surname;
+     private final byte[] password;
+     private final byte[] salt;
+     private final String email;
+//     private String phoneNumber;
+//
+//     private String addressLine1;
+//     private String addressLine2;
+//     private String city;
+//     private String postCode;
+//
+//     private Date dob;
+//
+//     private String gender;
+//     private String role;
+
+     public User(Integer ID, String firstName, String surname, String email, byte[] password, byte[] salt) {
+          this.ID = ID;
+          this.firstName = firstName;
+          this.surname = surname;
+          this.email = email;
+          this.password = password;
+          this.salt = salt;
+     }
+
+     public Integer getID() {
+          return ID;
+     }
+
+     public String getFirstName() {
+          return firstName;
+     }
+
+     public String getSurname() {
+          return surname;
+     }
+
+     public byte[] getPassword() {
+          return password;
+     }
+
+     public byte[] getSalt() {
+          return salt;
+     }
+
+     public String getEmail() {
+          return email;
+     }
+
+     @Override
+     public String toString() {
+          return "User{" +
+                  "ID=" + ID +
+                  ", firstName='" + firstName + '\'' +
```

Comment:
It took me far too long to realize you were escaping the *'* here haha. Does it not work to do *"'"*?

![image](https://user-images.githubusercontent.com/35970939/100521816-f1208980-319d-11eb-9400-8b9cf1eb2de7.png)

Reply:
I just let Intellij autogenerate these methods when I need them, haven't actually ever looked at the code output